### PR TITLE
LOG-4597: modify vector buffer and request to have mostly parity with…

### PR DESCRIPTION
**Pull request by jcantrill**
… fluentd

### Description
This PR:
* Adds buffer and request settings to all outputs for vector inorder to achieve  some parity to prior releases
* Modifies http api timeout to be an int instead of string of an int

### Links
* https://issues.redhat.com/browse/LOG-4597

cc @alanconway @syedriko

### CODE REVIEW

File: apis/logging/v1/output_types.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ type Http struct {
 
 	// Timeout specifies the Http request timeout in seconds. If not set, 10secs is used.
 	// +optional
-	Timeout string `json:"timeout,omitempty"`
+	Timeout int `json:"timeout,omitempty"`
```
Comment (alanconway) [inline apis/logging/v1/output_types.go:289]: Should be `TimeoutSeconds` for clarity. IIRC this is in the k8s API guidelines for time variables. 
Not a blocker if it's too late to change.

**Review by alanconway — CHANGES_REQUESTED** [review]

**Comment by alanconway** [comment]
/hold
Suggested a name change, otherwise all looks good.

File: apis/logging/v1/output_types.go (hunk 1 above)
Comment (jcantrill) [reply apis/logging/v1/output_types.go:289]: The http type was released for 5.7 so there is not a place to change it.  I have concerns about changing the time as-is

Comment (cahartma) [reply apis/logging/v1/output_types.go:289]: Can we release the hold?

**Pull request merged**